pkg/indexmanagement/scripts.go

```diff
@@ -50,8 +56,8 @@ r=json.load(sys.stdin)
 indices = [index for index in r if int(r[index]['settings']['index']['creation_date']) < $minAgeFromEpoc ]
 if "$writeIndex" in indices:
   indices.remove("$writeIndex")
-indices.sort()
-print ','.join(indices)
+for i in range(0, len(indices), 25):
+  print ', '.join(indices[i:i+25])
```

Comment:
I think the trailing space should be removed. Indices are expected to be joined by comma without spacing. See https://www.elastic.co/guide/en/elasticsearch/reference/current/multi-index.html

Reply:
BTW, shouldn't the join be implemented like `strings.Join(indices,",")` ? 
`import "strings" // is needed`